js/classes/entities/background/decoration.js

```diff
@@ -9,56 +9,97 @@ class Decoration {
+                },
+                TALL_PURPLE_FLOWER: {
+                    SPRITESHEET: "./sprites/flowers.png",
+                    SPRITESHEET_START_POS: new Vector(0, 32),
+                    FRAME_COUNT: 1,
+                    FRAME_DURATION: 1,
+                    SIZE: new Vector(16, 32),
+                    SCALE: 4
+                },
+                PRIDE_FLOWER: {
+                    SPRITESHEET: "./sprites/flowers.png",
+                    SPRITESHEET_START_POS: new Vector(0, 64),
+                    FRAME_COUNT: 1,
+                    FRAME_DURATION: 1,
+                    SIZE: new Vector(16, 32),
+                    SCALE: 4
+                },
+                CARROT: {
+                    SPRITESHEET: "./sprites/flowers.png",
+                    SPRITESHEET_START_POS: new Vector(0, 96),
+                    FRAME_COUNT: 1,
+                    FRAME_DURATION: 1,
+                    SIZE: new Vector(16, 32),
+                    SCALE: 4
+                },
+                POTATO: {
+                    SPRITESHEET: "./sprites/flowers.png",
+                    SPRITESHEET_START_POS: new Vector(0, 128),
+                    FRAME_COUNT: 1,
+                    FRAME_DURATION: 1,
+                    SIZE: new Vector(16, 32),
+                    SCALE: 4
+                }
             },
-            ChadHouse: {
-                SPRITESHEET: "./sprites/chad_house.png",
-                SPRITESHEET_START_POS: new Vector(0, 0),
-                FRAME_COUNT: 1,
-                FRAME_DURATION: 1,
-                SIZE: new Vector(192, 192),
-                SCALE: 3.5,
+            houses: {
+                BLACKSMITH_HOUSE: {
+                    SPRITESHEET: "./sprites/blacksmith_house.png",
+                    SPRITESHEET_START_POS: new Vector(0, 0),
+                    FRAME_COUNT: 4,
+                    FRAME_DURATION: 1/4,
+                    SIZE: new Vector(192, 192),
+                    SCALE: 3.5
+                },
+                CHAD_HOUSE: {
+                    SPRITESHEET: "./sprites/chad_house.png",
+                    SPRITESHEET_START_POS: new Vector(0, 0),
+                    FRAME_COUNT: 1,
+                    FRAME_DURATION: 1,
+                    SIZE: new Vector(192, 192),
+                    SCALE: 3.5
+                },
+                MAYOR_HOUSE: {
+                    SPRITESHEET: "./sprites/mayor_house.png",
+                    SPRITESHEET_START_POS: new Vector(0, 0),
+                    FRAME_COUNT: 1,
+                    FRAME_DURATION: 1,
+                    SIZE: new Vector(192, 192),
+                    SCALE: 3.5
+                }
             },
-            Tree: {
-                SPRITESHEET: "./sprites/tree.png",
-                SPRITESHEET_START_POS: new Vector(0, 0),
-                FRAME_COUNT: 1,
-                FRAME_DURATION: 1,
-                SIZE: new Vector(64, 64),
-                SCALE: 8,
+            trees: {
```

Comment:
Didn't realize you could make this so flexible by having trees and specific ones that stem from it. Is this JSON? I like it!